frontend/src/Components/Events.js

```diff
@@ -1,11 +1,10 @@
 import React from "react";
-// import { testData } from "../TestData.js";
-import { Event } from "./Event.js";
+import Event from "./Event.js";
 
 const Events = props => {
     return (
         <div className="events">
```

Comment:
From the list of events, you should be able to click on an event to find more info about that event.

Reply:
I agree with @dahfool, but I think we can do that as future work. For now I think we can move ahead with this.